packages/amplify-appsync-simulator/src/velocity/util/rds.ts

```diff
@@ -1,19 +1,32 @@
+const serializeRdsObject = rdsObject => {
+  return (rdsObject?.sqlStatementResults ?? []).map(statement =>
+    (statement?.records ?? []).map(record => {
+      const result = {};
+      record.forEach((row, index) => {
+        result[statement?.columnMetadata?.[index]?.name] = Object.values(row)?.[0];
```

Comment:
Hey @jariz, I know `toJsonString` also does not consider `null` values, but I think we could also update this in the same PR.

```
result[statement?.columnMetadata?.[index]?.name] = row['isNull'] || row['null'] ? null : Object.values(row)?.[0];
```
^ would account for both `{ "isNull": true, "null": true }`